pycookiecheat/pycookiecheat.py

```diff
@@ -64,23 +64,30 @@ def chrome_decrypt(encrypted_value: bytes, key: bytes, init_vector: bytes) \
     return clean(decrypted)
 
 
-def get_osx_config() -> dict:
-    """Get settings for getting Chrome cookies on OSX.
+def get_osx_config(browser_name: str = None) -> dict:
```

Comment:
Can we change the `browser_name` argument to just `browser`? I think that's clear enough.